### PR TITLE
DCS-198 Incidents by religion. DCS-199 Incidents by ethnic group.

### PR DESCRIPTION
**Pull request by bell-pa**
Add new fields `ethnicityCode` and `religionCode` to `/api/prisoners` and related resource responses. (`PrisonerDetail` class).
These primary key values are more stable than the description values used for `ethnicity` and `religion` and are easier to match. They will be used for new Use of Force reports - incidents by religious group and incidents by ethnic group. (DCS-198, DCS-199)

### CODE REVIEW

File: src/main/java/net/syscon/elite/api/model/PrisonerDetail.java [hunk 1]
```diff
@@ -24,73 +21,108 @@
...
     @NotNull
     private LocalDate dateOfBirth;
 
+    @ApiModelProperty(required = true, value = "The prisoner's gender.")
     @NotBlank
     private String gender;
 
+    @ApiModelProperty(required = true, value = "The prisoner's gender code.")
     @NotBlank
     private String sexCode;
```
Comment (andrewrlee) [inline src/main/java/net/syscon/elite/api/model/PrisonerDetail.java:60]: so they do it for this anyway which is good!

File: src/main/java/net/syscon/elite/api/model/PrisonerDetail.java [hunk 2]
```diff
@@ -24,73 +21,108 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @EqualsAndHashCode
+@Data
 public class PrisonerDetail {
     @JsonIgnore
     private Map<String, Object> additionalProperties;
 
+    @ApiModelProperty(required = true, value = "The prisoner's unique offender number (aka NOMS Number in the UK).")
```
Comment (andrewrlee) [inline src/main/java/net/syscon/elite/api/model/PrisonerDetail.java:29]: tend to put examples on these now..

File: src/main/java/net/syscon/elite/api/model/PrisonerDetail.java (hunk 1 above)
Comment (Mjwillis) [reply src/main/java/net/syscon/elite/api/model/PrisonerDetail.java:60]: Nice to give an example

File: src/main/java/net/syscon/elite/api/model/PrisonerDetail.java (hunk 2 above)
Comment (bell-pa) [reply src/main/java/net/syscon/elite/api/model/PrisonerDetail.java:29]: I've added examples for the religion, religionCode, ethnicity and ethnicityCode properties.

**Pull request merged**